oai_agents/agents/rl.py

```diff
@@ -52,7 +53,10 @@ def __init__(self, teammates_collection, args,
         self.use_policy_clone = use_policy_clone
 
         self.learner_type = learner_type
-        self.env, self.eval_envs = self.get_envs(env, eval_envs, deterministic, learner_type)
+        self.env, self.eval_envs = self.get_envs(env, eval_envs, deterministic, learner_type, start_timestep)
+        self.start_step = start_step
```

Comment:
It looks like you can get rid of 
self.start_step and just use  self.steps = start_step from the beginning so that we don't have to keep track of self.start_step

and can you give better names to these two variables or add comments to explain what they are

Reply:
So as far as i can tell steps is actually something like episodes but I didnt want to change it in case it effected a lot of other stuff like the wandb reporting since I remember there is both step and timestep and we use one and not the other